stylesheets/styles.css

```diff
@@ -336,10 +336,14 @@ td {
 }
 
 .horizontal {
-  float: center;
-  padding: .5em;
+padding: .5em;
```

Comment:
@AnkitKhatri55555 These two changes are different and also for two different files. Create a new branch, rebase and then create PR for the change in style.css .